etc/notebooks/test/test_plotting_libs.ipynb

```diff
@@ -385,7 +386,7 @@
    },
    "outputs": [],
    "source": [
-    "show(p)"
+    "handle = show(p, notebook_handle=True)"
```

Comment:
Does this have to do with changes on line 255, where we are no longer importing `show` from bokeh? Is this code using a different `show`?